src/components/NavBar.tsx

```diff
@@ -7,22 +6,36 @@ import {
   faTools,
 } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import tools from '../assets/tools.json';
 
 import './NavBar.css';
 import OfekLogo from '../assets/images/OfekLogo.png';
 
-interface PropsType {
-  toggleMainWidth(): void;
+interface NavBarPropsType {
+  toggleMainWidth:() => void;
   mini: Boolean;
 }
-const NavBar = (props: PropsType) => {
+const NavBar = ({ toggleMainWidth, mini }: NavBarPropsType) => {
   const SideBarElement: HTMLElement | null = document.getElementById('sidebar');
 
+  const pages =  [
+    {
+        link: "/about",
+        icon: faQuestionCircle,
+        display: "About"
+    },
+    {
+        link: "/pybuild",
+        icon: faHammer,
+        display: "PyBuild"
+    }
+  ];
+
   const toggleSideBar = (mouse: Boolean) => {
     if(mouse) {
-      props.toggleMainWidth();
+      toggleMainWidth();
       if (SideBarElement) {
-        if (props.mini) SideBarElement.style.width = '250px';
+        if (mini) SideBarElement.style.width = '250px';
```

Comment:
`SideBarElement.style.width = mini ? '250px' : '80px'`

Reply:
changed it